x-pack/test_serverless/functional/test_suites/apps/lens/group1/config.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { createTestConfig } from "../../../../config.base";
+
+
+export default createTestConfig({
+  serverlessProject: 'es',
+  testFiles: [require.resolve('../../../common'), require.resolve('.')],
```

Comment:
I'm not sure if this `common` test suite is truly required for all `test_serverless` test suites as done in all other suites.

Reply:
I just removed the `common` test suite as I don't see any reason for these tests.

Reply:
Removed `config.ts` in favor of top-level `common/config.ts`.